tests/integration/challenges/information_retrieval/test_information_retrieval_challenge_b.py

```diff
@@ -14,6 +14,9 @@
 CYCLE_COUNT = 3
 
 
+@pytest.mark.skip(
+    "This test uses the browse website functionality, which VCR doesn't support yet, this means it's slower, costs money and is less reliable. It will be unskipped once VCR support browse_website"
+)
```

Comment:
Why should `browse_website` rely on VCR? Requests to websites don't cost us any money, and with the right choice of content, we can trust that the website is static over time.

Reply:
yeah here this test uses content not static, there is some metadata apparently. I removed the skip so we can merge this PR and talk about that in a separate PR.